src/app/standup/goals/_partials/CounterDay/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { Box, Flex } from '@chakra-ui/react';
+
+import { useDarkMode } from '@/hooks/useDarkMode';
+
+interface CounterDayProps {
+  numberOfDay: number | string;
+  colorScheme?: string;
+}
+
+export const CounterDay = ({ numberOfDay, colorScheme }: CounterDayProps) => {
+  const { colorModeValue } = useDarkMode();
+
+  if (colorScheme === undefined || colorScheme === null) {
+    colorScheme = 'gray';
+  }
+
```

Comment:
This is avoidable if you give a default props to `colorScheme`. Also provide props types using FC.

```suggestion
export const CounterDay: FC<CounterDayProps> = ({ numberOfDay, colorScheme = 'gray' }) => {
  const { colorModeValue } = useDarkMode();

```